invenio_app_ils/ui/backoffice/src/pages/LoanDetails/components/LoanMetadata/LoanMetadata.scss

```diff
@@ -1,3 +0,0 @@
-.item-inline {
```

Comment:
empty file?

Reply:
I can't seem to find it on branch `29-backoffice-item`